manifests/init.pp

```diff
@@ -46,6 +46,7 @@
   Hash[String, Hash] $hosts = {},
   Variant[Array[String], Optional[String]] $includes = undef,
   String $config_comment = 'dhcpd.conf',
+  Optional[String] $shared_network = undef,
```

Comment:
Is it fine to allow empty strings here, or should we update it?
```suggestion
  Optional[String[1]] $shared_network = undef,
```

Reply:
I'd also like to document every new parameter. Ideally we'd have all parameters documented, but if we start with documenting new ones we at least move in that direction.